setup.py

```diff
@@ -27,20 +30,16 @@ def load_requirements(fn):
         return [x.rstrip() for x in list(f) if x and not x.startswith('#')]
 
 
-if use_deps:
-    ext_modules = [
-        Extension(
-            "pyndl.ndl_parallel",
-            ["pyndl/ndl_parallel.pyx"],
-            extra_compile_args=['-fopenmp'],
-            extra_link_args=['-fopenmp'],
-            include_dirs=[numpy.get_include()]
-        )
-    ]
-    cmdclass = {'build_ext': build_ext}
-else:
-    ext_modules = []
-    cmdclass = {}
+ext_modules = [
+    Extension(
+        "pyndl.ndl_parallel",
+        ["pyndl/ndl_parallel.pyx"],
+        extra_compile_args=['-fopenmp'],
+        extra_link_args=['-fopenmp'],
+        #include_dirs=[numpy.get_include()]  # this is dealt with in the boot strap class
```

Comment:
Is there a reason to keep this comment?

I think everyone looking in the diff will see that it is now done in the boot strap class or will not care about this comment.